app/sidekiq/scrape_keywords_job.rb

```diff
@@ -0,0 +1,10 @@
+# frozen_string_literal: true
+
+class ScrapeKeywordsJob
+  include Sidekiq::Job
+
+  def perform(keyword_ids)
```

Comment:
It would be best for the background job to process only one keyword. So it would result in rescheduling one job per keyword, thus providing isolation between each job and observability (i.e., one keyword job could fail and be retried independently from the others).